### PR TITLE
96 multilang improvements

### LINKED ISSUE #96: Multilang: other langs are not initialized
The proper behaviour (the one implemented for title and description for instance) should be:
- when creating a multilang object, the string provided for the current lang should be used as a default/initial value for all the offered languages;
- when updating the text, only the current lang should be updated.

At the moment:
- on creation, other langs are not populated
- on update, the value in the current lang is created, and other langs are removed
(issue replicable at least on ``other identifier / agent ``)

### PR DESCRIPTION
**Pull request by cezio**
fix #96

### CODE REVIEW

**Comment by etj** [comment]
Moved #96 to inprogress:
- creation of default entries for non-current languages work
- update will modify all the entries with the current one

**Comment by cezio** [comment]
@etj could you review?

File: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html [hunk 1]
```diff
@@ -68,13 +68,13 @@ <h3>{{ _('Dataset Metadata') }}</h3>
             <tr>
                 <th scope="row" class="dataset-label">{{ field.label }}</th>
                 <td class="dataset-details">
-                    <span style="font-weight:bold">{{_('OthId Identifier')}}: </span><span>{{ alt_id.identifier }}</span><br/>
+                    <span style="font-weight:bold">{{_('Other Identifier')}}: </span><span>{{ alt_id.identifier }}</span><br/>
```
Comment (etj) [inline ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html]: these i18n changes were not requested... some label are now mismatching due to the fuzzy completion 
Opening #118

File: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html [hunk 2]
```diff
@@ -68,13 +68,13 @@ <h3>{{ _('Dataset Metadata') }}</h3>
...
 
                         {% if alt_id.agent %}
-                    <span style="font-weight:bold">{{_('OthId Organization')}}: </span><span>{{ alt_id.agent.agent_name[lang] }}</span>
```
Comment (etj) [inline ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html]: #118

File: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html [hunk 3]
```diff
@@ -68,13 +68,13 @@ <h3>{{ _('Dataset Metadata') }}</h3>
...
+                    <span style="font-weight:bold">{{_('Other Organization')}}: </span><span>{{ alt_id.agent.agent_name[lang] }}</span>
                             {% if alt_id.agent.agent_identifier %}
                     <br/>
-                    <span style="font-weight:bold">{{_('OthId Organization ID')}}: </span><span>{{ alt_id.agent.agent_identifier }}</span><br/>
```
Comment (etj) [inline ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html]: #118

**Comment by etj** [comment]
Closing, included in #122

**Pull request closed without merging**